### PR TITLE
Website: update the title and description

### PR DESCRIPTION
**Pull request by tuhaihe**
---

## Change logs

Update the meta information for the Cloudberry Database website, and update the pull request template for contributors.

## Contributor's checklist

Here are some reminders before you submit your pull request:

* Make sure that your Pull Request has a clear title and commit message.
* Sign the Contributor License Agreement as prompted for your first-time contribution.
* Make sure that your changes deployment preview is successful.
* List your communications in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb-site/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (if has or needed).
* Feel free to ask for the @doc team or other people to help review and approve.

### CODE REVIEW

**Comment by tuhaihe** [comment]
Hi @TomShawn, PTAL.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -14,8 +14,9 @@ closes: #ISSUE_Number
 
 Here are some reminders before you submit your pull request:
 
-* Make sure that your Pull Request has a clear title and commit message.
-* Sign the Contributor License Agreement as prompted for your first-time contribution.
+* Make sure that your Pull Request has a clear title and commit message. You can take [git-commit](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) template as a reference.
+* Sign the Contributor License Agreement as prompted for your first-time contribution(*One-time setup*).
```
Comment (TomShawn) [inline .github/pull_request_template.md:18]:
```suggestion
* Sign the Contributor License Agreement as prompted for your first-time contribution (*One-time setup*).
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -14,8 +14,9 @@ closes: #ISSUE_Number
 
 Here are some reminders before you submit your pull request:
 
-* Make sure that your Pull Request has a clear title and commit message.
-* Sign the Contributor License Agreement as prompted for your first-time contribution.
+* Make sure that your Pull Request has a clear title and commit message. You can take [git-commit](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) template as a reference.
```
Comment (TomShawn) [inline .github/pull_request_template.md:17]:
```suggestion
* Make sure that your Pull Request has a clear title and commit message. You can take the [Git commit template](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) as a reference.
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -14,8 +14,9 @@ closes: #ISSUE_Number
...
+* Sign the Contributor License Agreement as prompted for your first-time contribution(*One-time setup*).
+* Learn the [code contribution](https://cloudberrydb.org/contribute/code) and [doc contribution](https://cloudberrydb.org/contribute/doc) guide for better collaboration.
```
Comment (TomShawn) [inline .github/pull_request_template.md:19]:
```suggestion
* Learn the [code contribution](https://cloudberrydb.org/contribute/code) and [doc contribution](https://cloudberrydb.org/contribute/doc) guides for better collaboration.
```

File: src/pages/index.js [hunk 4]
```diff
@@ -66,8 +66,8 @@ export default function Home() {
   const { siteConfig } = useDocusaurusContext();
   return (
     <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Learn about Cloudberry Database in just a few minutes."
+      title={`Next generation unified database for analytics and AI`}
+      description="Cloudberry Database ships with PostgreSQL 14.4 as the kernel, 100% open source, and helps you leverage the data value."
```
Comment (TomShawn) [inline src/pages/index.js:70]:
```suggestion
      description="Cloudberry Database ships with PostgreSQL 14.4 as the kernel. It is 100% open source and helps you leverage the value of your data."
```

File: src/pages/index.js [hunk 5]
```diff
@@ -66,8 +66,8 @@ export default function Home() {
   const { siteConfig } = useDocusaurusContext();
   return (
     <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Learn about Cloudberry Database in just a few minutes."
+      title={`Next generation unified database for analytics and AI`}
```
Comment (TomShawn) [inline src/pages/index.js:69]: If it is a title of a website, title case capitalization is better.
```suggestion
      title={`Next-Generation Unified Database for Analytics and AI"`}
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: src/pages/index.js (hunk 5 above)
Comment (tuhaihe) [reply src/pages/index.js:69]: Both ways are okay, so keep it unchanged. Thanks.

**Pull request merged**